src/Components/Todo/InputField.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { Box, Button, Grid, Input } from "@material-ui/core";
+import { connect, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { addTodo } from "../../redux/actions";
+import myStyles from "../../MaterialUI/myStyles";
+import InputFilter from "../inputFilter";
+
+const InputField = (props) => {
+  const { currentTasks } = props;
```

Comment:
You can destructure props in function argument on the fly and also avoid creating a new variable and save memory.

`const InputField = ({ currentTasks }) => {`